services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/settings.py

```diff
@@ -275,17 +275,19 @@ def _merge_resources_in_settings(
 
     # merge all resources
     empty_resource_entry: SimcoreServiceSettingLabelEntry = (
-        SimcoreServiceSettingLabelEntry(
-            name="Resources",
-            setting_type="Resources",
-            value={
-                "Limits": {"NanoCPUs": 0, "MemoryBytes": 0},
-                "Reservations": {
-                    "NanoCPUs": 0,
-                    "MemoryBytes": 0,
-                    "GenericResources": [],
+        SimcoreServiceSettingLabelEntry.parse_obj(
+            dict(
+                name="Resources",
+                type="Resources",
```

Comment:
-Q: why are you not using the constructor?

-TIP: an alternative is to  add ``Config.allow_population_by_field_name = True`` to ``SimcoreServiceSettingLabelEntry``

Reply:
I think that was some very old code. I now Know we should go for the parsing instead of the constructor. I've seen that using the constructor does not trigger the validators.